starboard/tools/testing/test_runner.py

```diff
@@ -862,6 +906,11 @@ def main():
       "--dry_run",
       action="store_true",
       help="Specifies to show what would be done without actually doing it.")
+  arg_parser.add_argument(
+      "-e",
+      "--extended",
+      help="Name of executable target, which to run, using extended interface"
```

Comment:
Do we need a separate entry point and flag for this ? Can't we simply keep using `--target_name` and call the extended interface if it's available ?

Reply:
Removed `--extended` argument. But now `test_runner` returns underlying code only when exactly one `--target_name` is specified. In other cases the behaviour is unchanged.